Modix.Bot/Modules/ReplModule.cs

```diff
@@ -150,19 +147,20 @@ private async Task<EmbedBuilder> BuildEmbedAsync(IGuildUser guildUser, Result pa
         {
             var returnValue = parsedResult.ReturnValue?.ToString() ?? " ";
             var consoleOut = parsedResult.ConsoleOut;
-            var status = string.IsNullOrEmpty(parsedResult.Exception) ? "Success" : "Failure";
+            var exception = !string.IsNullOrEmpty(parsedResult.Exception);
+            var status = exception ? "Failure" : "Success";
 
             var embed = new EmbedBuilder()
                     .WithTitle($"REPL Result: {status}")
-                    .WithColor(string.IsNullOrEmpty(parsedResult.Exception) ? Color.Green : Color.Red)
+                    .WithColor(exception ? Color.Red : Color.Green)
                     .WithUserAsAuthor(guildUser)
                     .WithFooter(a => a.WithText($"Compile: {parsedResult.CompileTime.TotalMilliseconds:F}ms | Execution: {parsedResult.ExecutionTime.TotalMilliseconds:F}ms"));
 
             embed.WithDescription(FormatOrEmptyCodeblock(parsedResult.Code, "cs"));
 
-            if (parsedResult.ReturnValue != null)
+            if (parsedResult.ReturnValue != null && parsedResult.ReturnTypeName != null)
```

Comment:
Not sure if adding this check really gives us any benefit

Reply:
More than likely not, I'll remove it